js/conditionals.js

```diff
@@ -0,0 +1,214 @@
+"use strict";
+
+/**
+ * TODO:
+ * Write some JavaScript that uses a `confirm` dialog to ask the user if they
+ * would like to enter a number. If they click 'Ok', prompt the user for a
+ * number, then use 3 separate alerts to tell the user:
+ *
+ * - whether the number is even or odd
+ * - what the number plus 100 is
+ * - if the number is negative or positive
+ *
+ * if what the user enters is not a number, use an alert to tell them that, and
+ * do *not* display any of the above information.
+ *
+ * Can you refactor your code to use functions?
+ */
+
+function messageForEvenOrOdd(number) {
+    var message;
+    if (number % 2 === 0) {
+        message = "This is an even number!";
+    } else {
+        message = "This is an odd number!";
+    }
+    return message;
+}
+
+function plusOneHundMessage(number) {
+    return "This number plus 100 is " + (number + 100) + "!";
+}
+function mathGame() {
+    var go = confirm("Hey, want to do some math?");
+    if (go === true) {
+        var number = parseInt(prompt("Give me a number"));
+        if (isNaN(number)) {
```

Comment:
Recommend putting together parseFloat and isNaN to be an isNumeric function that returns a boolean

Reply:
I have created an isNumeric function that will return a boolean value after parsing its parameters and running !isNaN(). Now our if runs on the happy path of being a number, while else displays an alert informing the user they have not entered a number.